gremlin-core/src/test/java/org/apache/tinkerpop/gremlin/jsr223/ImportGremlinPluginTest.java

```diff
@@ -24,13 +24,16 @@
 import org.apache.tinkerpop.gremlin.util.Gremlin;
 import org.junit.Test;
 
+import com.google.common.collect.ImmutableSet;
+
+import java.lang.reflect.Field;
 import java.lang.reflect.Method;
 import java.util.Collections;
 import java.util.stream.Collectors;
 import java.util.stream.Stream;
 
 import static org.hamcrest.MatcherAssert.assertThat;
-import static org.hamcrest.core.IsCollectionContaining.hasItems;
+import static org.hamcrest.Matchers.*;
```

Comment:
could you please remove the wildcard here (it's not our code style to allow that - all imports are explicit)? thanks.

Reply:
Done.